docker-compose.replica-set.yml

```diff
@@ -81,6 +81,7 @@ services:
       - SQL_DB_NAME
       - SQL_DB_LOGGING_ENABLED
       - ACBS_DURABLE_FUNCTIONS_LOG_DELETION_SCHEDULE
+      - DELETION_AUDIT_LOGS_DELETE_AFTER_SECONDS
```

Comment:
Think this doesnt need to be in other yml files since transactions work work not on a replica set. Need to remember to add to github actions once the flag is removed

Reply:
Note to self to add CHANGE_STREAM_ENABLED to TFM-api